WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/sections/insights/usecases/FollowersUseCase.kt

```diff
@@ -84,17 +84,17 @@ class FollowersUseCase(
         forced: Boolean,
         fetchMode: PagedMode
     ): State<Pair<FollowersModel, FollowersModel>> {
-        runBlocking(mainDispatcher) {
```

Comment:
with context should do the same thing and wait for the result before it continues with the next step.

Reply:
I believe this change slightly changes the beahvior. I can't tell if it causes any issues but I thought it'd be better to comment on it to make sure we are on the same page.

If we use "runBlocking" we can be sure the uiState will be updated before any other ui event is handled (assuming the caller is on UI thread). If we use "withContext" we can still be sure the caller doesn't perform any other action until this whole method completes. However, we can't be sure if the next UI event is handled before the uiState is updated or not. Wdyt?